frontend/public/index.html

```diff
@@ -14,12 +14,6 @@
       href="https://api.mapbox.com/mapbox-gl-js/v2.1.1/mapbox-gl.css"
       rel="stylesheet"
     />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
-      rel="stylesheet"
-    />
```

Comment:
If you want to include Montserrat in the project, then you probably need to include its license as well: https://www.fontsquirrel.com/license/montserrat

Reply:
Also, looks like there're some conflicts with: https://github.com/CollActionteam/clothing-loop/pull/127/files